morphia/src/test/java/org/mongodb/morphia/TestIdField.java

```diff
@@ -96,12 +96,12 @@ public void testMapAsId() throws Exception {
     @Entity
```

Comment:
Trying to translate the key values to reflect the document names vs the java field names brought this issues to the surface.